cmd/loadtest/output.go

```diff
@@ -103,7 +105,12 @@ func printBlockSummary(c *ethclient.Client, bs map[uint64]blockSummary, startNon
 		p.Printf("Transactions per sec: %v\n", number.Decimal(tps))
 		p.Printf("Gas Per Second: %v\n", number.Decimal(gaspersec))
 		p.Printf("Latencies - Min: %v\tMedian: %v\tMax: %v\n", number.Decimal(minLatency.Seconds()), number.Decimal(medianLatency.Seconds()), number.Decimal(maxLatency.Seconds()))
-		// TODO: Add some kind of indication of block time variance
+		p.Printf("Mean Blocktime: %vs\n", number.Decimal(meanBlocktime))
+		p.Printf("Median Blocktime: %vs\n", number.Decimal(medianBlocktime))
+		p.Printf("Minimum Blocktime: %vs\n", number.Decimal(minBlocktime))
+		p.Printf("Maximum Blocktime: %vs\n", number.Decimal(maxBlocktime))
+		p.Printf("Blocktime Standard Deviation: %vs\n", number.Decimal(stddevBlocktime))
+		p.Printf("Blocktime Variance: %vs\n", number.Decimal(varianceBlocktime))
```

Comment:
I wonder it it makes sense to show like 2 decimal places here? It seems like it will usually be 0s or 1s most of the time?

Reply:
Hmm I agree that the standard deviation and variance will likely be 0 most of the time, but if it does increase due to some fluctuations in block time (configured as 12s) but actually like `10s, 11s, 12s, 14s, 12s, 12s` the standard deviation and variance from these samples will more likely be decimals (1.77, and 1.33 respectively in the above example).

Maybe it will rarely be fully useful in most cases though..